fla/ops/log_linear_attn/naive.py

```
@@ -0,0 +1,50 @@
+
+def construct_H_matrix(a, L):
+    T = a.size(-1)
+    A = torch.exp(segsum(a))
+    H = torch.zeros_like(A)
+    for level in range(int(np.log2(T)) + 1):
+        mask = construct_level_mask(level, L)
+        H += A * mask
+    return H
+
+
+def naive_log_linear_attn(q, k, v, g, l):
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)